sdk/storage/Azure.Storage.Blobs/samples/Sample03a_Batching.cs

```diff
@@ -137,38 +149,45 @@ public void FineGrainedBatching()
         [Test]
         public void BatchErrors()
         {
-            // Get a connection string to our Azure Storage account.
             string connectionString = ConnectionString;
+            string containerName = Randomize("sample-container");
+
+            #region Snippet:SampleSnippetsBatch_Troubleshooting
+            // Get a connection string to our Azure Storage account.
+            //@@ string connectionString = "<connection_string>";
+            //@@ string containerName = "sample-container";
 
             // Get a reference to a container named "sample-container" and then create it
             BlobServiceClient service = new BlobServiceClient(connectionString);
-            BlobContainerClient container = service.GetBlobContainerClient(Randomize("sample-container"));
+            BlobContainerClient container = service.GetBlobContainerClient(containerName);
             container.Create();
-            try
-            {
-                // Create a blob named "valid"
-                BlobClient valid = container.GetBlobClient("valid");
-                valid.Upload(new MemoryStream(Encoding.UTF8.GetBytes("Valid!")));
 
-                // Get a reference to a blob named "invalid", but never create it
-                BlobClient invalid = container.GetBlobClient("invalid");
+            // Create a blob named "valid"
+            BlobClient valid = container.GetBlobClient("valid");
+            valid.Upload(new MemoryStream(Encoding.UTF8.GetBytes("Valid!")));
+
+            // Get a reference to a blob named "invalid", but never create it
+            BlobClient invalid = container.GetBlobClient("invalid");
 
-                // Delete both blobs at the same time
-                BlobBatchClient batch = service.GetBlobBatchClient();
+            // Delete both blobs at the same time
+            BlobBatchClient batch = service.GetBlobBatchClient();
+            try
+            {
                 batch.DeleteBlobs(new Uri[] { valid.Uri, invalid.Uri });
             }
-            catch (AggregateException ex)
-            {
-                // An aggregate exception is thrown for all the indivudal failures
+
+            //@@ catch (AggregateException ex)
```

Comment:
Why are we commenting this out?

Reply:
Let's just remove the verification code

Reply:
But then why have it as a test?

Reply:
We'll still know if the exception ever changes.  I care more about the readability right now and this is hard to read for a sample.  We can make the test better after releasing.